### ISSUE TITLE
"Imprisonment" config Easter Egg song?

### ISSUE THREAD

**Issue by func-octo** [Zet0rz/nzombies#320]
I know you are already working on this, and idk if you had a song in mind, but 115 is a good one for nZombies. I mean I guess it sounds insane, but does get adrenaline flowing.

**Comment by Zet0rz** [comment]
115 is used in Breakout. This needs a different one

**Comment by func-octo** [comment]
Found this. Might help: [https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwiB74Lgp4_QAhWGQCYKHdohBQUQtwIIHDAA&url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3D8JTXUPDGncM&usg=AFQjCNGiiPGeXP0l49GCrrId7p64DfD0EQ](url)

**Comment by minin43** [comment]
PSA: 115 legitimately sucks ass. Please use ANY of the other easter egg songs for configs. Every map has a secret song, yet EVERYONE uses the shitty 115 song. IMO, the Avenged Sevenfold songs are way better, but I'd just ask everyone stay away from 115.

**Comment by Zet0rz** [comment]
That's personal opinion, I always liked Elena's better than Avenge Sevenfold's, but it won't be 115 as that's already used. I'm thinking I might go "Where are we Going?" since it's kinda Mob of the Dead-themed this map

**Comment by Zet0rz** [comment]
It's gonna have a full easter egg from the get-go btw

**Comment by func-octo** [comment]
What if you mixed it up a bit and added an L4D/L4D2 song in there? Might work well, but I haven't heard any of them, so idk.

**Comment by Zet0rz** [comment]
Probably gonna pick this, I think it fits the theme the best
https://youtu.be/jz31BVNSWFE

Sticking to actual Zombies songs, or at least very very similarly themed ones

**Comment by func-octo** [comment]
I like it.

**Comment by minin43** [comment]
As long as she isn't screaming her heart out, I otherwise kind of like her voice. 115 just gets on my nerves every time I hear it.

**Comment by func-octo** [comment]
> As long as she isn't screaming her heart out, I otherwise kind of like her voice.

xD. Has anyone made a no-vocal version of 115?

**Comment by Zet0rz** [comment]
Config and Map Script added in dev branch, along with navmesh. You guys mind beta testing it? I'd also like to know if you can solve the EE, if it's too easy maybe, and how many rounds it took you to do so. It's doable in solo.

**Comment by minin43** [comment]
Getting spammed this error: 

```
[ERROR] addons/nzombies-development/gamemodes/nzombies/entities/entities/nz_activatable.lua:142: attempt to call method 'IsActivatable' (a nil value)
  1. v - addons/nzombies-development/gamemodes/nzombies/entities/entities/nz_activatable.lua:142
   2. Run - lua/includes/modules/hook.lua:84
    3. unknown - gamemodes/base/gamemode/cl_init.lua:75
```

**Comment by Zet0rz** [comment]
That function is defined in /gamemode/enemies/sh_ent_ext.lua, check that and see if it is there. If not, try redownloading it or just copy pasting it into your file.

**Comment by minin43** [comment]
But I literally just downloaded the latest version of the dev branch

**Comment by minin43** [comment]
It's still being spammed, and I redownloaded the dev version.

I'm also only receiving the crowbar on game start.

**Comment by Zet0rz** [comment]
Sounds related maybe lol?
The function shouldn't error if it is defined. I currently play just fine on current dev branch version. Maybe you didn't do a clean install in case some files were deleted (and left over from your previous installation)?

**Comment by minin43** [comment]
I've downloaded it twice now with the same result each time. The only difference is I have to go through the weapon and m. box and perk replacement, you probably aren't, and that's where the error is coming from.

**Comment by Zet0rz** [comment]
Just did a playtest with friends, having them hunt for the main easter egg. We ended up beating the map in 17 rounds, but with a lot of hints given by me. Notes to self about things I should improve/tweak:
- Fix electrical boxes' LightningAura not triggering properly when outside PVS
- Fix Office lights sometimes setting light 1 to red after all are lit up
- Add PVS to game win camera so the main core doesn't disappear if away from that area when another player triggers it's destruction
- Change Workbench model so it's not an error for half of everyone playing
- Fix max ammo breaking when a player has a shield

**Comment by func-octo** [comment]
'PVS'?

**Comment by lolleko** [comment]
Potential Visibility Set.

Description from [VDC](https://developer.valvesoftware.com/wiki/PVS):

A Potentially Visible Set (or Potential Visibility Set) is a collection of visleaves which might be visible from a given location. It is not a precise measure of visibility but thanks to being pre-compiled into a map is a very fast one. It forms the basis for many run-time tests, including networking filtering and rendering control.

TL;DR: Not everything of the map and entities is rendered & networked for the client. This exists for performance and security reasons.
